package.xml

```diff
@@ -32,6 +32,8 @@
   <depend>assimp</depend>
   <depend>liboctomap-dev</depend>
   <depend>eigenpy</depend>
```

Comment:
So, we still depend on eigenpy then? Is there such a thing as package variants on ROS?

Reply:
for here and now, yes, so that we can have a CI testing both eigenpy and nanoeigenpy
On the ros2 release repo, I'll patch this to have only the good dependencies.

Reply:
Perfect, thanks!